packages/bruno-electron/src/index.js

```diff
@@ -1,5 +1,4 @@
 const path = require('path');
-const isDev = require('electron-is-dev');
```

Comment:
Does that mean the dependency can be removed from package.json?

Reply:
I'm all for maintaining a clean `package.json`. 

Removed the now (in `packages/bruno-electron`) redundant package and updated the lockfile.
It seems like there's more changes in the lockfile than I expected:
- `node_modules/@n8n/vm2` and `@n8n/vm2` weren't tracked and now are
- `packages/bruno-electron` received a version bump
- a bunch of "dev": true lock flags disappeared - I honestly don't know how that relates to my change